README.md

```diff
@@ -884,7 +884,10 @@ The following standard options are available (see http://facebook.github.io/reac
 - `selectionState`
 - `textAlign`
 - `textAlignVertical`
-- `underlineColorAndroid`
+- ~~`underlineColorAndroid`~~
+
+`underlineColorAndroid` is not supported now on `t-comb-form-native` due to sometimes cause crashes on Android, especially on ScrollView. See more on:
```

Comment:
`tcomb-form-native`

Please replace also "due to sometimes". Better "due to random crashes".

Reply:
@alvaromb you can merge now this PR